### PR TITLE
CORE-5774 Remove cli package plugin

### PR DESCRIPTION
**Pull request by kyriathar**
Removes cli package plugin as it is being ported to `corda-runtime-os` repo with [#1666](https://github.com/corda/corda-runtime-os/pull/1666).

### CODE REVIEW

**Review by vkolomeyko — APPROVED** [review]
LGTM

**Comment by josephzunigadaly** [comment]
Please don't merge this until we have merged the runtime-os side and sent out communication to the team

**Comment by josephzunigadaly** [comment]
We are ready to merge now

**Pull request merged**
